PyInstaller/loader/pyimod02_archive.py

```diff
@@ -268,23 +267,33 @@ def _import_aesmod(self):
 
         PyCrypto 2.4 and 2.6 uses different name of the AES extension.
         """
-        # Not-so-easy way: at bootstrap time we have to load the module from the
-        # temporary directory in a manner similar to pyi_importers.CExtensionImporter.
-        from pyimod03_importers import CExtensionImporter
-        importer = CExtensionImporter()
-        # NOTE: We _must_ call find_module first.
         # The _AES.so module exists only in PyCrypto 2.6 and later. Try to import
         # that first.
         modname = 'Crypto.Cipher._AES'
-        mod = importer.find_module(modname)
-        # Fallback to AES.so, which should be there in PyCrypto 2.4 and earlier.
-        if not mod:
-            modname = 'Crypto.Cipher.AES'
+
+        if sys.version_info[0] == 2:
+            # Not-so-easy way: at bootstrap time we have to load the module from the
+            # temporary directory in a manner similar to pyi_importers.CExtensionImporter.
+            from pyimod03_importers import CExtensionImporter
+            importer = CExtensionImporter()
+            # NOTE: We _must_ call find_module first.
             mod = importer.find_module(modname)
+            # Fallback to AES.so, which should be there in PyCrypto 2.4 and earlier.
             if not mod:
-                # Raise import error if none of the AES modules is found.
-                raise ImportError(modname)
-        mod = mod.load_module(modname)
+                modname = 'Crypto.Cipher.AES'
+                mod = importer.find_module(modname)
+                if not mod:
+                    # Raise import error if none of the AES modules is found.
+                    raise ImportError(modname)
+            mod = mod.load_module(modname)
+        else:
+            kwargs = dict(fromlist=['Crypto', 'Cipher'])
+            try:
+                mod = __import__(modname, **kwargs)
+            except ImportError:
+                modname = 'Crypto.Cipher.AES'
+                mod = __import__(modname, **kwargs)
+
```

Comment:
How is this change related to the commit? Look like this is fixing some other issue. If so, please put it into a commit of its own

Reply:
The Python 2 code uses the `CExtensionImporter` to import `Crypto.Cipher.AES`; Python 3 can't, since the extension can now be imported using the standard Python import mechanism. Hence this change.

I put this in the same commit, since these two changes (removing the `CExtensionImporter` and removing code which uses `CExtensionImporter`) are related.